.github/workflows/storybook-check.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     check:
         runs-on: ubuntu-latest
         if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
+        timeout-minutes: 60
```

Comment:
Curious why this is so high. Looking at the workflow runs so far, these usually complete under 20 minutes with a few outliers around 25 minutes. Would a more conservative value of `30` or `40` work here?